assets/css/style.css

```diff
@@ -123,6 +121,48 @@ a {
   width: 200px; /*PLACEHOLDER*/
 }
 
+
+/*Home Review Section*/
+.home-review-section {
```

Comment:
The grid could use a margin. Everything butts right up against the edges. Also, some padding at the bottom to separate it from the next section on larger screen sizes.